db/020_schema_nw.sql

```diff
@@ -54,6 +54,37 @@ CREATE INDEX ON nw.link USING BTREE(i_node);
 CREATE INDEX ON nw.link USING BTREE(j_node);
 CREATE INDEX ON nw.link USING GIST(geom);
 
+CREATE VIEW nw.view_link_oneway AS (
+  SELECT
+    link_id,
+    1::smallint AS link_dir,
+    i_node,
+    j_node,
+    oneway,
+    length_m,
+    link_modes,
+    link_label,
+    data_source,
+    source_date,
+    geom
+  FROM nw.link
+  UNION
+  SELECT
+    link_id,
+    -1::smallint AS link_dir,
+    j_node AS i_node,
+    i_node AS j_node,
+    oneway,
+    length_m,
+    link_modes,
+    link_label,
+    data_source,
+    source_date,
+    ST_Reverse(geom) AS geom
+  FROM nw.link
+  WHERE oneway IS true
```

Comment:
Fix this to `false`

Reply:
... `WHERE NOT oneway`

Reply:
Consider wrapping this to a CTE block, adding a "pseudo id" as `row_number()` and using it in pgrouting